core-web/apps/dotcms-ui/src/app/view/components/dot-device-selector/dot-device-selector.component.ts

```diff
@@ -72,10 +76,12 @@ export class DotDeviceSelectorComponent implements OnInit, OnChanges {
                     if (this.disabled) {
                         this.placeholder = 'No devices';
                     }
+                    this.cd.detectChanges();
                 },
                 () => {
                     this.disabled = true;
                     this.placeholder = 'No devices';
+                    this.cd.detectChanges();
```

Comment:
We can add a third callback (complete), so we can add `this.cd.detectChanges()` once.

```suggestion
      },
      () => {
          this.disabled = true;
          this.placeholder = 'No devices';
       },
      () =>  this.cd.detectChanges()
   )
```

Reply:
added